011-kafka-canary.md

```diff
@@ -0,0 +1,40 @@
+# Strmixi Canary
+
+Implement a Kafka canary tool which will act as an indicator of whether Kafka clusters are operating correctly. This would be achieved by creating a canary topic and continously producing and consuming events on the topics. 
```

Comment:
Please explain what continuously means. There is not continuous messaging - it will be in intervals and it is important to understand what kind of intervals are you talking about.